WooCommerce/Classes/ViewRelated/Keyboard/KeyboardStateProvider.swift

```diff
@@ -8,7 +8,17 @@ struct KeyboardState: Equatable {
     let isVisible: Bool
     /// The frame of the keyboard when it is fully shown or hidden.
```

Comment:
nit: a single newline before comment block can improve readability

Reply:
Thanks. I'm never sure about the spacing of these properties. 😅 Added a newline in 9a01ed36bcb8147a9dd066ccfa0f980b7e6286c0.